src/main.rs

```diff
@@ -1394,14 +1403,85 @@ async fn list_deadline(config: Config, args: &ListDeadline) -> Result<String, Er
 
 // // --- CONFIG ---
 
-async fn config_check_version(_args: &ConfigCheckVersion) -> Result<String, Error> {
-    match cargo::compare_versions(None).await {
-        Ok(Version::Latest) => Ok(format!("Tod is up to date with version: {VERSION}")),
-        Ok(Version::Dated(version)) => Err(Error::new(
-            "cargo",
-            &format!("Tod is out of date with version: {VERSION}, latest is:{version}"),
-        )),
-        Err(e) => Err(e),
+async fn config_check_version(
+    args: &ConfigCheckVersion,
+    mock_url: Option<String>,
+) -> Result<String, Error> {
+    match compare_versions(mock_url).await {
+        Ok(Version::Latest) => {
+            let msg = format!("Tod is up to date with version: {VERSION}");
+            if args.force {
+                return Ok(msg);
+            }
+            println!("{msg}");
+            Ok(String::new())
```

Comment:
```suggestion
            Ok(msg)
```

Does this basically do the same thing? When returning `Ok(String)` the string should be printed to the terminal before exiting.